model/TestRunnerService.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Created by Antoine on 27/01/15
+ * at 15:42
+ */
+
+namespace oat\taoTestLinear\model;
```

Comment:
Could you add the licence at the beginning of the file with copyright
Could you add the phpdoc of method added